doc/example_odMLs/ex_1.rdf

```diff
@@ -1,142 +1,114 @@
-        <odml:Section rdf:resource="https://g-node.org/projects/odml-rdf#Section"/>
-        <odml:type>crew</odml:type>
-        <odml:definition>Information on the crew</odml:definition>
-        <odml:name>s12</odml:name>
-        <odml:hasSection rdf:resource="https://g-node.org/projects/odml-rdf#18bb5a7e-2097-43f2-a782-e6f612107e93"/>
-      </rdf:Description>
-    </odml:hasSection>
-    <odml:definition>Information on the crew</odml:definition>
-  </rdf:Description>
-  <rdf:Description rdf:about="https://g-node.org/projects/odml-rdf#0f96a050-2d9b-498f-a532-fbfcc6aae55e">
-    <odml:Section rdf:resource="https://g-node.org/projects/odml-rdf#Section"/>
-    <odml:type>crew</odml:type>
-    <odml:hasProperty>
-      <rdf:Description rdf:about="https://g-node.org/projects/odml-rdf#7b0572c1-fcfc-4010-aebf-730397ac29af">
-        <odml:definition>List of crew members names</odml:definition>
-        <odml:dtype>person</odml:dtype>
-        <odml:Property rdf:resource="https://g-node.org/projects/odml-rdf#Property"/>
-        <odml:hasValue>
-          <rdf:Description rdf:nodeID="4818d863-9abe-4ee0-9ec0-0bcaa36f2df1">
-            <rdf:li>Zaphod Beeblebrox</rdf:li>
-            <rdf:li>Ford Prefect</rdf:li>
-            <rdf:li>Arthur Philip Dent</rdf:li>
-            <rdf:li>Tricia Marie McMillan</rdf:li>
-          </rdf:Description>
-        </odml:hasValue>
-        <odml:name>NameCrewMembers</odml:name>
-      </rdf:Description>
-    </odml:hasProperty>
-    <odml:hasSection rdf:resource="https://g-node.org/projects/odml-rdf#1d6469b5-5322-408b-916e-3224b7a22008"/>
-    <odml:definition>Information on the crew</odml:definition>
-    <odml:name>TheCrew</odml:name>
-    <odml:hasProperty rdf:resource="https://g-node.org/projects/odml-rdf#298938a6-0996-4a92-982b-837a08a1a220"/>
-  </rdf:Description>
-  <rdf:Description rdf:about="https://g-node.org/projects/odml-rdf#18bb5a7e-2097-43f2-a782-e6f612107e93">
-    <odml:type>crew</odml:type>
-    <odml:Section rdf:resource="https://g-node.org/projects/odml-rdf#Section"/>
-    <odml:reference>s21</odml:reference>
-    <odml:name>s121</odml:name>
-    <odml:definition>Information on the crew</odml:definition>
-  </rdf:Description>
-  <rdf:Description rdf:about="https://g-node.org/projects/odml-rdf#bc6cc466-5769-4da1-b90e-8e5595903a4a">
-    <odml:name>s2</odml:name>
-    <odml:hasSection rdf:resource="https://g-node.org/projects/odml-rdf#4b2d0e9d-6352-4676-97d7-fc0a795e83a9"/>
-    <odml:definition>Information on the crew</odml:definition>
-    <odml:Section rdf:resource="https://g-node.org/projects/odml-rdf#Section"/>
-    <odml:type>crew</odml:type>
-  </rdf:Description>
-</rdf:RDF>
+@prefix odml: <https://g-node.org/projects/odml-rdf#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+odml:hub1 odml:Hub odml:Hub ;
```

Comment:
I probably missed it in the last PR, but I think [this line here](https://github.com/G-Node/python-odml/commit/81cc66c20d20629c7051b0d6243832c5795213a3#diff-fc2680be278b7e90cfbbdcb32f792693L40) in the `rdf_converter` should be an `RDF.type` as predicate.